utility-belt/src/main/java/io/confluent/admin/utils/cli/TopicEnsureCommand.java

```diff
@@ -99,17 +103,17 @@ public static void main(String[] args) {
     } catch (ArgumentParserException e) {
       if (args.length == 0) {
         parser.printHelp();
-        success = true;
+        validationSuccess = true;
       } else {
         parser.handleError(e);
-        success = false;
```

Comment:
Since it defaults to `false`, no need to set to false -- just keep the one line that sets it to `true` when it's valid.

Reply:
I will make the change. thanks!!